build: `make deploy` fails on macOS Big Sur 11.1

```
% make deploy
Making all in src
Making all in doc/man
make[1]: Nothing to be done for `all'.
make[1]: Nothing to be done for `all-am'.
build-aux/install-sh -c -d Bitcoin-Qt.app/Contents/MacOS
STRIPPROG="/usr/bin/strip" /bin/sh /Users/hebasto1/bitcoin/build-aux/install-sh -c -s  ./src/qt/bitcoin-qt Bitcoin-Qt.app/Contents/MacOS/Bitcoin-Qt
/usr/local/bin/python3.9 ./contrib/macdeploy/macdeployqtplus Bitcoin-Qt.app Bitcoin-Core -translations-dir= -dmg
Traceback (most recent call last):
  File "/Users/hebasto1/bitcoin/./contrib/macdeploy/macdeployqtplus", line 22, in <module>
    from ds_store import DSStore
ModuleNotFoundError: No module named 'ds_store'
make: *** [Bitcoin-Core.dmg] Error 1
```

--- a/doc/build-osx.md
+++ b/doc/build-osx.md
@@ -30,6 +30,11 @@ If you want to build the disk image with `make deploy` (.dmg / optional), you ne
 brew install librsvg
 ```
 
+and [`macdeployqtplus`](../contrib/macdeploy/README.md) dependencies:
+```shell
+pip3 install ds_store mac_alias
+```
+
 The wallet support requires one or both of the dependencies ([*SQLite*](#sqlite) and [*Berkeley DB*](#berkeley-db)) in the sections below.
 To build Bitcoin Core without wallet, see [*Disable-wallet mode*](#disable-wallet-mode).
 

